plugins/microservices/src/atomic_apply_acl_operations.cpp

```diff
@@ -126,6 +127,26 @@ auto plugin_factory() -> irods::ms_table_entry*
 /// \retval 0        On success.
 /// \retval non-zero On failure.
 ///
+/// \b Example
+/// \code{.py}
+/// # The collection to apply the ACL operations to.
+/// *lpath = '/tempZone/home/alice';
```

Comment:
does a trailing slash matter here? is it worth noting if it DOES matter?

Reply:
It's not supposed to matter, but I bet the current implementation of the API doesn't handle it. Will give it a try.

I don't think it's worth mentioning. The user shouldn't need to worry about trailing slashes. We have a function which takes care of that.

Reply:
Confirmed the API doesn't handle trailing slashes.

Will write up and fix as part of this PR.

Reply:
Well, I'll write an issue.

Reply:
Created #7892.

Resolving since that is out-of-scope of this PR.